main.py

```diff
@@ -770,9 +887,72 @@ def clean_up_db(self):
             if len(verification[email]['verified']) == 1 and len(verification[email]['unverified']) >= 1:
                 db.delete(verification[email]['unverified'])
 
+class CronHandler(BaseHandler):
+    def get(self, job): # On url invoke
+        if job == "lunch":
+            update_lunch.read_lunches()
+            self.response.write("Success")
+
+class PrivacyHandler(BaseHandler): # Change and view privacy settings
+    def load_obj(self):
+        id = self.check_id()
+        if id is None:
+            return None
+
+        logging.info(id)
+
+        email = convert_id_to_email(id)
+
+        logging.info(email)
+
+        user_obj_query = db.GqlQuery("SELECT * FROM User WHERE email = :1", email)
+        for user_obj in user_obj_query:
+            return user_obj
+
+    def unicode_to_boolean(self, string):
+        if (string == 'true'):
+            return True
+        elif (string == 'false'):
+            return False
+        return None
+
+    def get(self):
+        user_obj = self.load_obj()
+        if user_obj is None:
+            self.error(403)
+            return 
+
+        response = {"share_photo": user_obj.share_photo, "share_schedule": user_obj.share_schedule}
+        self.response.write(json.dumps(response))
+
+    def post(self):
+        user_obj = self.load_obj()
+        if user_obj is None:
+            self.error(403)
+            return
+
+        logging.info(self.unicode_to_boolean(self.request.get('share_photo')))
+        logging.info(self.unicode_to_boolean(type(self.request.get('share_photo'))))
+
+        user_obj.share_photo = self.unicode_to_boolean(self.request.get('share_photo'))
+        user_obj.share_schedule = self.unicode_to_boolean(self.request.get('share_schedule'))
+        user_obj.put()
+        self.response.write(json.dumps({}))
+
+class AvatarHandler(BaseHandler):
+    def get(self, user):
+        id = self.check_id()
+        if id != "4093":
+            self.error(403)
+            return
+            
+        args = string.split(user, '_')
+        url = self.gen_photo_url(args[1], args[0], 'school_photos')
+        self.redirect(url)
 app = webapp2.WSGIApplication([
     ('/', MainHandler),
     ('/about', AboutHandler),
+    ('/avatar/(\w+).jpg', AvatarHandler),
```

Comment:
Um, what is this? We can't allow users to access any arbitrary photo at /avatar/foo_bar.jpg